Dockerfile

```diff
@@ -1181,10 +1181,14 @@ echo "Cleaning logs every $EVERY seconds"
 
 while true; do
   echo "Trimming airflow logs to ${RETENTION} days."
-  find "${DIRECTORY}"/logs \
+ 
+  files=$(find "${DIRECTORY}"/logs \
     -type d -name 'lost+found' -prune -o \
-    -type f -mtime +"${RETENTION}" -name '*.log' -print0 | \
-    xargs -0 rm -f
+    -type f -mtime +"${RETENTION}" -name '*.log' || true)
+ 
+  for file in ${files}; do
```

Comment:
This will not work if there are spaces in file names (which might happen) better to keep -print0 / xargs -0 and ignore errors there.

Reply:
I was not aware spaces on file names are a possibility (does not happen on my case)
Sure I thought of just ignoring errors on xargs rm, but I was afraid it would not solve all cases when the error appears on the find command.
I will do the change though, it's better than nothing.